src/output/icons.rs

```diff
@@ -200,9 +201,10 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     ".tcshrc"             => Icons::SHELL,          // 󱆃
     ".viminfo"            => Icons::VIM,            // 
     ".vimrc"              => Icons::VIM,            // 
-    ".Xauthority"         => Icons::CONFIG,         // 󱁻
-    ".xinitrc"            => Icons::CONFIG,         // 󱁻
-    ".Xresources"         => Icons::CONFIG,         // 󱁻
+    ".Xauthority"         => Icons::XORG,           // 
+    ".xinitrc"            => Icons::XORG,           // 
+    ".Xresources"         => Icons::CONFIG,         // 
```

Comment:
Shouldn't be `Icons::XORG` on `.Xresources`?

Also you may want add the `xorg.conf` file.

Also I don't know if any of these files (and similar) should also have the X.org logo.

 /etc/X11
└──  xorg.conf.d
   ├──  00-keyboard.conf
   ├──  10-amdgpu.conf
   └──  30-touchpad.conf


 ~/.config/xsettingsd
└──  xsettingsd.conf

Reply:
`xorg.conf.d` is a folder. `00-keyboard.conf` , `10-amdgpu.conf`  and `30-touchpad.conf` are not fixed; they depend on the system.

Reply:
Yeah, the dir already have an icon, and the files even if the names are kind-of generic could be the case that are different on different systems.

Reply:
`Yes` file's name can be different and `Yes` it is **kind of generic** but  if `00-keyboard.conf` is added, it can start with `00` or `10` or other. It becomes messy when you add all the combinations of all files.

Reply:
Good, as of your latest changes I think it's OK.